jme3-core/src/main/resources/Common/MatDefs/Light/PBRLighting.frag

```diff
@@ -208,19 +210,24 @@ void main(){
     gl_FragColor.rgb = vec3(0.0);
     vec3 ao = vec3(1.0);
 
-    #ifdef LIGHTMAP
+    #if defined(LIGHTMAP) || ( defined(AO_PACKED_IN_MR_MAP) && defined(USE_PACKED_MR) )
        vec3 lightMapColor;
-       #ifdef SEPARATE_TEXCOORD
-          lightMapColor = texture2D(m_LightMap, texCoord2).rgb;
-       #else
-          lightMapColor = texture2D(m_LightMap, texCoord).rgb;
+        #ifdef AO_PACKED_IN_MR_MAP
+            lightMapColor = metallicRoughnessAoValue.rrr; //ao value packed in red channel of MR map 
+        #else
+            #ifdef SEPARATE_TEXCOORD    
+                lightMapColor = texture2D(m_LightMap, texCoord2).rgb;
+            #else
+                lightMapColor = texture2D(m_LightMap, texCoord).rgb;
+            #endif                                                 
        #endif
-       #ifdef AO_MAP
-         lightMapColor.gb = lightMapColor.rr;
-         ao = lightMapColor;
+       #if defined(AO_MAP) || defined(AO_PACKED_IN_MR_MAP)
+            lightMapColor.gb = lightMapColor.rr;
+            ao = lightMapColor;
        #else
-         gl_FragColor.rgb += diffuseColor.rgb * lightMapColor;
+            gl_FragColor.rgb += diffuseColor.rgb * lightMapColor;
        #endif
+       
```

Comment:
I think the new code should go here:

https://github.com/jMonkeyEngine/jmonkeyengine/blob/master/jme3-core/src/main/resources/Common/MatDefs/Light/PBRLighting.frag#L227

And just replace (or multiply) the value of `vec3 ao`, so that it doesn't interfere with the lightmap code that can be used together with the ao.